Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -207,7 +208,10 @@ private class LoginCursorDataSource: NSObject, UITableViewDataSource {
         }
 
         let titleForSectionAtIndex = sectionTitles[section]
-        let logins = cursor?.filter { $0?.hostname.asURL?.host?.uppercaseString.startsWith(titleForSectionAtIndex) ?? false }
-        return logins?.flatMap { $0 } ?? []
+        let logins = cursor?.filter { $0?.hostname.asURL?.baseDomain()?.uppercaseString.startsWith(titleForSectionAtIndex) ?? false }
+        let flattenLogins = logins?.flatMap { $0 } ?? []
+        return flattenLogins.sort { login1, login2 in
+            login1.hostname.asURL?.baseDomain() < login2.hostname.asURL?.baseDomain()
```

Comment:
Can we also sort the subdomains in the cases where the base domains are equal? For instance, if I had `hangouts.google.com`, `mail.google.com`, and `www.google.com`, I'd expect to see them in that order.